src/pages/signin/SignInPageLayout/Footer.js

```diff
@@ -211,6 +229,9 @@ const Footer = (props) => {
 
 Footer.propTypes = propTypes;
 Footer.displayName = 'Footer';
+Footer.defaultProps = {
+    scrollViewRef: {current: null},
```

Comment:
default props are declared at top of file for all components in the codebase.

@allroundexperts let's follow the same conventions